vignettes/query-RESULTS-silviculture-data-using-bcdata.Rmd

```diff
@@ -0,0 +1,215 @@
+
+Silviculture data for British Columbia are largely collected and stored through the [Reporting Silviculture Updates and Land Status Tracking System (RESULTS) database](https://www2.gov.bc.ca/gov/content?id=F62992DA5B324446AD5E1B5BFFA404CC)---including information on the management of openings, disturbances, silviculture activities and obligation declarations as required by the [Forest and Range Practices Act](https://www2.gov.bc.ca/gov/content?id=D65006C0326B4E0F838194044C10CA86). [RESULTS data sets are publically available through the B.C. Data Catalogue](https://catalogue.data.gov.bc.ca/dataset?q=RESULTS+silviculture) under the [Open Government Licence - British Columbia](https://www2.gov.bc.ca/gov/content?id=A519A56BC2BF44E4A008B33FCF527F61).
+
+
+## A Silviculture Data Exploration&mdash;Reforestation with Climate Considerations
+
+[Western larch](https://en.wikipedia.org/wiki/Larix_occidentalis) has been identified as a tree species that will be well-adapted to projected future climates in northern British Columbia.  In 2013, forest policy was developed that allowed foresters to plant western larch outside of its natural range as a climate change adaptation silviculture measure. 
+
+**How much western larch has been planted in the Prince George Natural Resource District?**  
+
+Let's use the `bcdata` package to search, retrieve and explore the RESULTS silviculture data and answer this question. 
+
+
+### Getting Started
+
+To start, let's load the `bcdata` package. We will also load the `dplyr` and `ggplot2` packages to help us work with the data. You can learn more about the `dplyr` package [here](https://dplyr.tidyverse.org/) and the `ggplot2` package [here](https://ggplot2.tidyverse.org/):
+
+```{r libs, eval = FALSE}
+library(bcdata)
+library(dplyr)
+library(ggplot2)
+```
+
+
+### Getting the Data
+
+We can gather the data we need to answer our question using the [RESULTS - silviculture forest cover dataset](https://catalogue.data.gov.bc.ca/dataset/258bb088-4113-47b1-b568-ce20bd64e3e3). First, let's take a look at the metadata record using `bcdc_get_record()`:
+
+```{r get-record}
+# Get the metadata using the human-readable record name
+bcdc_get_record("results-forest-cover-silviculture")
+```
+
+We see that this is a [Web Service-enabled](https://www2.gov.bc.ca/gov/content?id=95D78D544B244F34B89223EF069DF74E) geospatial data set, so we can query and retrieve this geospatial data set using `bcdc_query_geodata()`:
+
+```{r examine-silv-layer}
+# Query the data using the permanent ID of the record to guard against name changes
+bcdc_query_geodata("258bb088-4113-47b1-b568-ce20bd64e3e3")
+```
+
+This query shows that this data set has over 850,000 features and 159 fields.  Each feature is a treatment unit within a harvested opening, and contains information on the leading five tree species that are present in each treatment unit, including stems per hectare, age, and height.
```

Comment:
I wonder if we can generalize this to _over_ 150 fields or something so that if it changes, this will still be true.